scripts/release/utils.injector.js

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env node
+import json2md from 'json2md';
+import fs from 'node:fs';
+import path from 'node:path';
+
+import { Build } from '../../documents/scripts/paths.js';
+import { error, errorHandler, success } from '../helpers/index.js';
+import { ELEMENT_DIST, ELEMENT_SOURCE, generateDocList } from './util.js';
+
+/*
+ * Return remove and replace unwanted text for json2md.
+ */
+const json2mdTrim = (text) => {
+  return text?.replace('\n', ' ').replaceAll('\r', ' ').replaceAll('`', "'");
+};
+
+/**
+ * Return full text comment from Signature Typedoc structure.
+ * @param {signature} signature typedoc's Signature
+ * @returns {string} text comment
+ */
+const getComment = (signature) => {
+  if (signature?.comment?.summary && signature.comment.summary.length <= 0) {
+    return '';
+  }
+  return signature.comment.summary.map((item) => item.text).join('');
```

Comment:
add `const summary = signature?.comment?.summary;` could reduce a lot of repetitions.

Reply:
Fixed.